pkg/image/registryclient/credentials.go

```diff
@@ -0,0 +1,86 @@
+package registryclient
```

Comment:
You're copy&pasting this (at least parts) from `pkg/image/importer/credentials.go` but I don't see that file (those bits) being removed?